newIDE/app/resources/examples/isometric-path-finding/README.md

```diff
@@ -0,0 +1 @@
+This shows how to have an object move automatically and avoid obstacles in an isometric game.
```

Comment:
Thanks for the example game 👍👍

Reply:
I put every thing in an external layout because it's easier to copy/paste to make other levels to test, but it may be unsettling to a new comer.